docs/development_tips.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Tips and Tricks"
+---
+
+This page provides some tips and tricks that may help you as a contributor to Home Assistant. The list here is by no means exhaustive, and if you pick up any additional undocumented tips and tricks, please open a PR to add them here.
+
+## Tips and Tricks
+
+### Keep PRs simple
+
+See the [Component Checklist](creating_component_code_review#5-make-your-pull-request-as-small-as-possible) for PR expectations.
+
+### Test package dependency changes in Home Assistant
+
+See the [API library docs](api_lib_index#trying-your-library-inside-home-assistant) for more information.
+
+### Test Core integration changes in your production Home Assistant environment
+
+If made a change to a core integration and you want to test it in your production Home Assistant environment, copy the integration folder into `/config/custom_components`, add a version to the `manifest.json` (e.g. "version": "0.0"), and restart Home Assistant. Home Assistant will always prioritize integrations in `custom_components` over the core integration. Don't forget to remove it once you are done testing otherwise you will be stuck on that version.
```

Comment:
```suggestion
If you made a change to a core integration and you want to test it in your production Home Assistant environment, copy the integration folder into `/config/custom_components`, add a version to the `manifest.json` (e.g. "version": "0.0"), and restart Home Assistant. Home Assistant will always prioritize integrations in `custom_components` over the core integration. Don't forget to remove it once you are done testing otherwise you will be stuck on that version.
```